packages/core/src/modules/connections/ConnectionsModule.ts

```diff
@@ -67,6 +69,8 @@ export class ConnectionsModule {
     this.messageSender = messageSender
     this.didResolverService = didResolverService
     this.registerHandlers(dispatcher)
+
+    this.registerFeatures(featureRegistry)
```

Comment:
What if we call this from within the register method of the module? That way it is more aligned with modules as we'll have them in 0.3.0. In 0.3.0 I would also like to move handler registration to the module

Reply:
Absolutely. I think it should be there in order to enforce feature registration when defining a module (and also the handlers, as you mentioned). As I said in a previous comment, I've done some tries by using the register method (had to add featureRegistry as parameter) but didn´t like so much. Maybe we can improve this module API to add separate methods such as:

- registerDependencies(dependencyManager)
- registerFeatures(featureManager)
- registerHandlers(dispatcher)